applications/fishing-map/src/features/datasets/datasets.hook.ts

```diff
@@ -20,16 +26,30 @@ import {
 } from './datasets.slice'
 
 export const useNewDatasetConnect = () => {
-  const dataviews = useSelector(selectContextAreasDataviews)
+  const contextDataviews = useSelector(selectContextAreasDataviews)
+  const enviromentalDataviews = useSelector(selectEnvironmentalDataviews)
   const { upsertDataviewInstance } = useDataviewInstancesConnect()
-  const usedColors = dataviews?.flatMap((dataview) => dataview.config?.color || [])
 
   const addNewDatasetToWorkspace = useCallback(
-    (dataset) => {
-      const dataviewInstance = getContextDataviewInstance(dataset.id, usedColors)
-      upsertDataviewInstance(dataviewInstance)
+    (dataset: Dataset) => {
+      let dataviewInstance
+      const datasetType =
+        dataset.configuration?.propertyToInclude && dataset.configuration?.propertyToIncludeRange
+          ? 'enviromental'
```

Comment:
There is an [enum defined for `DatasetTypes`](https://github.com/GlobalFishingWatch/frontend/blob/a9e66b1c44f33892904339582479772e472a0a3a/packages/api-types/src/datasets.ts#L28), should it be used here?

Reply:
This is just a workaround in the frontend to distinguish between environmental layer and context areas datasets and it is not related with the datasetType. So to make it clear I've renamed it to `layerType`